internal/storage/sql/testing/testing.go

```diff
@@ -210,6 +194,44 @@ func Open() (*Database, error) {
 	}, nil
 }
 
+func newMigrator(db *sql.DB, driver fliptsql.Driver) (*migrate.Migrate, error) {
+	var (
+		dr  database.Driver
+		err error
+	)
+
+	switch driver {
+	case fliptsql.SQLite:
+		dr, err = sqlite3.WithInstance(db, &sqlite3.Config{})
+	case fliptsql.Postgres:
+		dr, err = pg.WithInstance(db, &pg.Config{})
+	case fliptsql.CockroachDB:
+		dr, err = cockroachdb.WithInstance(db, &cockroachdb.Config{})
+	case fliptsql.MySQL:
+		dr, err = ms.WithInstance(db, &ms.Config{})
+
+		// https://stackoverflow.com/questions/5452760/how-to-truncate-a-foreign-key-constrained-table
+		if _, err := db.Exec("SET FOREIGN_KEY_CHECKS = 0;"); err != nil {
+			return nil, fmt.Errorf("disabling foreign key checks: %w", err)
+		}
```

Comment:
It seems we're doing the same thing with MySQL that we do with SQLite here. Is that correct?

If so, should we now move this into the `Drop` command in the migrator alongside SQLite?